components/IndoorDirections/IndoorScenario.js

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from 'react';
+import { G } from 'react-native-svg';
+import { HallXCoordinates } from '../../constants/HallXCoordinates';
+import { ClassGraph } from '../../constants/ClassGraph';
+import { DifferentFloorDirections } from './TypesOfDirections/DifferentFloorDirections';
+import { SameFloorDirections } from './TypesOfDirections/SameFloorDirections';
+import { DifferentBuildingDirections } from './TypesOfDirections/DifferentBuildingDirections';
+
+export function IndoorScenario(props) {
```

Comment:
Personally, I would have called this function _IndoorDirections_(not sure if it wasn't called before). The coder doesn't need to know that the component is a scenario and at first, I thought that component was one particular indoor scenario, not all scenarios.

Reply:
Will do!

Reply:
🥇 💯